chacractl/util.py

```diff
@@ -20,6 +21,7 @@ def load_config():
     config = get_config_path()
     if not config:
         return
+    sys.modules.pop('chacractl', None)
```

Comment:
Ok cool, `conftest.py` with the following works the way I need it to work:

```
import sys
import pytest


@pytest.fixture(autouse=True)
def clear_config(request):
    sys.modules.pop('chacractl', None)
```

I will close this and open a new PR.